DOCS/man/options.rst

```diff
@@ -4395,6 +4395,17 @@ The following video options are currently all specific to ``--vo=gpu`` and
     ``--temporal-dither`` is in use. 1 (the default) will update on every video
     frame, 2 on every other frame, etc.
 
+``--gpu-deint-shader=<mode>``
```

Comment:
Option name and value names probably still need to be bikeshedded. It's also possible we simply want to hook it up with ``--deinterlace``.